packages/fuel-indexer-tests/tests/trybuild.rs

```diff
@@ -135,6 +153,73 @@ module:
             TestKind::Fail,
             manifest_content.clone(),
         ),
+        (
+            "pass_if_using_sway_amm_abi.rs",
+            "sway_amm.yaml",
+            TestKind::Pass,
+            manifest_with_contract_abi("AMM-contract-abi.json"),
+        ),
+        (
+            "pass_if_using_sway_dao_abi.rs",
+            "sway_dao.yaml",
+            TestKind::Pass,
+            manifest_with_contract_abi("DAO-contract-abi.json"),
+        ),
+        (
+            "pass_if_using_sway_asset_contract_abi.rs",
+            "asset_contract.yaml",
+            TestKind::Pass,
+            manifest_with_contract_abi("asset-contract-abi.json"),
+        ),
+        (
+            "pass_if_using_sway_distributor_contract_abi.rs",
+            "distributor_contract.yaml",
+            TestKind::Pass,
+            manifest_with_contract_abi("distributor-contract-abi.json"),
+        ),
+        (
+            "pass_if_using_sway_escrow_contract_abi.rs",
+            "escrow_contract.yaml",
+            TestKind::Pass,
+            manifest_with_contract_abi("escrow-contract-abi.json"),
+        ),
+        (
+            "pass_is_using_sway_exchange_contract_abi.rs",
+            "exchange_contract.yaml",
+            TestKind::Pass,
+            manifest_with_contract_abi("exchange-contract-abi.json"),
+        ),
+        // NOTE: I don't think the ABI tokens are being properly generated from this contract JSON
```

Comment:
I will ask the Sway team and the SDK team why this particular contract presents an issue. There is a simple struct `EvmAddress` that for whatever reason is just returned in the ABI tokens (so on the SDK side)